Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1979,13 +1979,21 @@ extension BrowserViewController: LegacyTabDelegate {
             let creditCardHelper = CreditCardHelper(tab: tab)
             tab.addContentScript(creditCardHelper, name: CreditCardHelper.name())
 
-            creditCardHelper.foundFieldValues = { fieldValues in
+            creditCardHelper.foundFieldValues = { fieldValues, type in
                 guard let tabWebView = tab.webView as? TabWebView else { return }
+                guard let type = type else { return }
+
+                switch type {
+                case .formInput:
+                    tabWebView.accessoryView.reloadViewFor(.creditCard)
+                    tabWebView.reloadInputViews()
+                case .formSubmit:
+                    // Ref: FXIOS-6111
+                    // Action will be added to present a half sheet
+                    // for remember or update the credit card
+                    break
```

Comment:
@petruSt24, when this gets merged, you should use this case to show the flow for saving a credit card. You already built the UX but this is where it gets shown.

Let me know if you have any questions